microsoft-edge/extensions-chromium/index.md

```diff
@@ -102,6 +102,17 @@ To submit your extension to the store, you need to provide the following items:
 
 After you've successfully submitted your extension, your extension undergoes a review process and either passes or fails the certification process.  Owners are notified of the outcome and given next steps as required.  If you submit an extension update to the store, it begins a new review process.
 
+### Featured badge in Edge Add-ons Store
+
+Featured extensions on Microsoft Edge Add-ons Store follow the Microsoft’s recommended best practices related to security, privacy, transparency, and user experience. Microsoft Edge Add-ons Store team reviews each extension for adherence to EAS best practices before granting it a “Featured” badge. 
+
+## Disclaimer
+
```

Comment:
```suggestion
```
heading adds no info, is not needed, negative toned, and the section flows better without it.

gh rendered:
https://github.com/MicrosoftDocs/edge-developer/blob/v-bpappu/msedgeaddons/microsoft-edge/extensions-chromium/index.md